python/ray/llm/_internal/serve/deployments/routers/router.py

```diff
@@ -310,10 +313,18 @@ def _get_configured_serve_handle(self, model_id: str):
     async def _get_response(
         self,
         *,
-        body: Union[CompletionRequest, ChatCompletionRequest, EmbeddingRequest],
+        body: Union[
```

Comment:
similarly let's use overload decorator for this method to avoid listing all the possible combinations of types